components/automate-ui/src/app/page-components/create-user-modal/create-user-modal.component.ts

```diff
@@ -135,10 +142,8 @@ export class CreateUserModalComponent implements OnInit, OnDestroy {
   }
 
   hasAttemptedInputWithError(field: string): boolean {
-    return ((this.createUserForm.get(field).hasError('required') &&
-      this.createUserForm.get(field).touched) ||
-      (this.createUserForm.get(field).hasError('pattern') &&
```

Comment:
You deleted `pattern` here (and that's good, because it is redundant with the `minLength` check) but you also need to make the corresponding change in the Validator itself, i.e., delete this line: https://github.com/chef/automate/blob/2e5e0ec16d4bda3b3f8b5f1e601181bc71a4fa27/components/automate-ui/src/app/page-components/create-user-modal/create-user-modal.component.ts#L49

Reply:
I'm going to keep the NON_BLANK validator in since it prevents passwords that are all spaces. However, it wasn't triggering any error alerts so I also updated the `field is required` error conditional so it will display for all space values.

Reply:
That's even better! 😁